comments-by-guideline-and-success-criterion.md

```diff
@@ -78,7 +78,34 @@ This applies directly as written, and as described in [Intent from Understanding
 <div class="note">Content that is only used on hardware that is fixed in place OR that has no sensor to detect or change the orientation is covered under the essential exception and not required to provide support for orientation changes.</div>
 
 ##### identify-input-purpose
-<p class="ednote">This section is to be developed by the WCAG2ICT Task Force.</p>
+
+ **1.3.5: Identify Input Purpose:**
```

Comment:
I don't think we need to add in the original heading or the original SC text. This will be handled when we get the includes working.